content/ja/docs/reference/access-authn-authz/rbac.md

```diff
@@ -1087,7 +1092,7 @@ RBACRoleBindingを使用して、permissive ABACポリシーを複製できま
 {{< warning >}}
 次のポリシーでは、**すべて**のサービスアカウントがクラスター管理者としてふるまうことを許可しています。
 コンテナで実行されているアプリケーションは、サービスアカウントのクレデンシャルを自動的に受け取ります。
-secretsの表示や権限の変更など、APIに対して任意のアクションを実行できます。
+secretの表示や権限の変更など、APIに対して任意のアクションを実行できます。
```

Comment:
Is it better for using capital character?
```suggestion
Secretの表示や権限の変更など、APIに対して任意のアクションを実行できます。
```

Reply:
Thanks.
I will update.